app_builder/bootstrapper.py

```diff
@@ -21,21 +21,28 @@ def _load_config():
         "linux_appbuilder_path": settings.get("linux_appbuilder_path") or "",
         "win_node_name": "node",
         "win_appbuilder_name": "appbuilder",
-        "required_appbuilder_cli_version": "2.7.3"
+        "min_required_appbuilder_cli_version": "2.8.0", #inclusive
+        "max_allowed_appbuilder_cli_version": "2.9.0"  #exclusive
     }
 
 def _verify_appbuilder_cli_version(installed_appbuilder_cli_version):
     global _has_compatible_working_appbuilder_cli
-    required_appbuilder_cli_version = parse_version_string(get_config("required_appbuilder_cli_version"))
-    if installed_appbuilder_cli_version == required_appbuilder_cli_version:
+    min_required_appbuilder_cli_version = get_config("min_required_appbuilder_cli_version")
+    max_allowed_appbuilder_cli_version = get_config("max_allowed_appbuilder_cli_version")
+    validMinVersion = match(installed_appbuilder_cli_version, ">="+min_required_appbuilder_cli_version)
+    validMaxVersion = match(installed_appbuilder_cli_version, "<"+max_allowed_appbuilder_cli_version)
+    if validMinVersion and validMaxVersion:
         _has_compatible_working_appbuilder_cli = True
         log_info("Telerik AppBuilder has been initialized successfully")
     else:
         _has_compatible_working_appbuilder_cli = False
-        log_fail("Cannot load the Telerik AppBuilder package because the required version of Telerik AppBuilder command-line interface is {required_version}.\n".
-            format(required_version=".".join(required_appbuilder_cli_version)) +
-            "The Telerik AppBuilder command-line interface version that has been found is {installed_version}".
-            format(installed_version=".".join(installed_appbuilder_cli_version)))
+        if not validMinVersion:
+            log_fail("Cannot load the Telerik AppBuilder package because the version of Telerik AppBuilder command-line interface should be at least {required_min_version}.\n".
+                format(required_min_version=min_required_appbuilder_cli_version) +
+                "The Telerik AppBuilder command-line interface version that has been found is {installed_version}".
+                format(installed_version=installed_appbuilder_cli_version))
+        if not validMaxVersion:
+            log_fail("You should update your Telerik AppBuilder Sublime package.\n")
```

Comment:
1. -> You have updated your Telerik AppBuilder package to {installed_version}. To be able to load it, you need to update your Telerik AppBuilder CLI to {required_min_version}.x.
2. -> You have updated your Telerik AppBuilder CLI to {installed_version}. To be able to load the Telerik AppBuilder package, you need to update it to {required_sublime_package_version}.